packages/consumption/src/consumption/CoreErrors.ts

```diff
@@ -268,6 +268,10 @@ class Requests {
         return new CoreError("error.consumption.requests.invalidRequestItem", message);
     }
 
+    public noMatchingRelationship(address: string) {
+        return new CoreError("error.consumption.requests.noMatchingRelationship", `An active or pending Relationship with the given address '${address}' does not exist.`);
+    }
```

Comment:
we have to completely re-think this error and maybe introduce multiple new ones:

- for the IncomingRequestController we accept no relationship and an active relationship - these are the things we can answer to
- for the OutgoingRequestController I think this is fitting

Two possibilities to fix this:
1. provide two errors describing the two different things the controllers need
2. pass the complete message to the method

I'd highly vote for 2. but the error code has to be improved significantly. 

It should describe what does `noMatchingRelationship` mean? Not matching for what operation?

Reply:
if you want to answer the question about the operation inside error code, you're in option 1 - I also think option 2 fits better to the existing errors. I think I wouldn't even mention the correct statuses in the message because of the special cases, just the current wrong status.

Reply:
Great!